packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -125,9 +125,14 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
       isStickyBottom !== nextState.isStickyBottom ||
       this.props.stickyPosition !== nextProps.stickyPosition ||
       this.props.children !== nextProps.children ||
-      _isHeightOrWidthDifferent(this._nonStickyContent, this._placeHolder) ||
-      _isHeightOrWidthDifferent(this._nonStickyContent, this._stickyContentTop) ||
-      _isHeightOrWidthDifferent(this._nonStickyContent, this._stickyContentBottom)) as boolean;
+      (this._nonStickyContent &&
```

Comment:
Could you break this up into variables? It's very difficult to read as is.

Reply:
done